practice.md

```diff
@@ -88,6 +88,24 @@ When an 18F project has a non-standard license status (e.g. it's a fork of a pre
 
 All 18F team members are expected to make new source code repositories public from the time of creation. This means we often publish drafts in our repos that may change substantially. If you're interested in learning more about the contents of a repo, email 18F@gsa.gov and we'll direct you to the right person or team.
 
+#### Protecting sensitive information
+
+As part of responsibly working in the open, 18F team members are expected to protect information that needs to be protected. We already get training and guidance about information we can’t publish for ethical, legal, communications, and security reasons — this section is just a reminder about sensitive information (formally called “controlled unclassified information”) to especially carefully protect when working with our open source projects. Sensitive information can be code, configuration, content, or documentation.
+
+If 18F team members aren't sure whether they should make something public, they can ask a person on our 18F Infrastructure team for advice. (Try asking in #compliance, but ask a general question instead of pasting the actual information, since we can't put sensitive information in Slack.)
+
+Sensitive information we need to protect includes:
+
+* Information an attacker could plausibly use to help them compromise a live system. Examples:
```

Comment:
Sounds like there's enough ambiguity in how people use "live" that this could cause some confusion for other people too. How about we remove "live", so that this becomes "Information an attacker could plausibly use to help them compromise our systems." ?